packages/fxa-settings/src/components/PageSettings/index.tsx

```diff
@@ -13,18 +13,38 @@ import LinkedAccounts from '../LinkedAccounts';
 import * as Metrics from '../../lib/metrics';
 import { useAccount } from '../../models';
 import { DeleteAccountPath } from 'fxa-settings/src/constants';
-import { Localized } from '@fluent/react';
+import { Localized, useLocalization } from '@fluent/react';
 import DataCollection from '../DataCollection';
 
 export const PageSettings = (_: RouteComponentProps) => {
   const { uid } = useAccount();
+  const { l10n } = useLocalization();
 
   Metrics.setProperties({
     lang: document.querySelector('html')?.getAttribute('lang'),
     uid,
   });
   Metrics.usePageViewEvent(Metrics.settingsViewName);
 
+  // Temp test to see how often we actually display fallback text
+  useEffect(() => {
+    Metrics.logViewEvent(Metrics.settingsViewName, 'test.fallback.start');
+    const ftlId = 'app-footer-mozilla-logo-label';
+    const text = l10n.getString(ftlId);
+
+    if (text && text !== ftlId) {
+      Metrics.logViewEvent(
+        Metrics.settingsViewName,
+        'test.fallback.text-not-needed'
+      );
+    } else {
+      Metrics.logViewEvent(
+        Metrics.settingsViewName,
+        'test.fallback.text-needed'
+      );
+    }
+  }, [l10n]);
```

Comment:
I see why you added [l10n], but a comment might be good here.

Reply:
Heh I actually only included it in the dep array since it was required to. `[]` just runs it once of course which is what I wanted but it complained at me and told me to add `l10n` to the dep array. I am not sure if that is worth a comment 🤔